### PR TITLE
Enhancements to overall App UI Design

### PR DESCRIPTION
**Pull request by Vishnuxx**
I've been working on some updates to the app's user interface design. Here's a summary of what I've done:

*  Enhanced the overall app user interface design to improve user experience and visual appeal.
* Rewrote some CSS styles in Svelte to use Tailwind CSS.

I've attached a screenshot of the new design to this pull request. Please take a look and let me know your thoughts!

Screenshot 2024-03-22 at 1 35 03 AM

Looking forward to your feedback and suggestions.

### CODE REVIEW

File: ui/package-lock.json
Comment (Israel-Laguan) [inline ui/package-lock.json:1]: As you didn't changed `package.json`, changes on this file are not needed. Please consider removing this change

File: ui/src/components/BrowserWidget.svelte [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <div id="browser-content" class="flex-grow overflow-auto">
     {#if $agentState?.browser_session.screenshot}
       <img
-        class="browser-img"
+        class=" block object-contain w-full"
```
Comment (Israel-Laguan) [inline ui/src/components/BrowserWidget.svelte:25]: NIT:
Extra space here

File: ui/bun.lockb
Comment (Israel-Laguan) [inline ui/bun.lockb:1]: Not sure if this is needed, please double check

File: ui/src/components/MessageContainer.svelte [hunk 2]
```diff
@@ -13,10 +13,10 @@
   });
 </script>
 
-<div id="message-container" class="flex-grow overflow-y-auto pr-2" bind:this={messageContainer}>
+<div id="message-container" class="flex-grow overflow-y-auto px-4" bind:this={messageContainer}>
   {#if $messages !== null}
     {#each $messages as message}
-      <div class="flex items-start space-x-3 mt-4">
+      <div class={`flex items-start p-3 space-x-3 mt-4   ${message.from_devika ? "" : "bg-[#161b1e]"}`}>
```
Comment (Israel-Laguan) [inline ui/src/components/MessageContainer.svelte:19]: NIT: 
extra space
```suggestion
      <div class={`flex items-start p-3 space-x-3 mt-4 ${message.from_devika ? "" : "bg-[#161b1e]"}`}>
```

File: ui/src/components/MessageContainer.svelte [hunk 3]
```diff
@@ -64,7 +64,7 @@
             </div>
           {:else}
             <div
-              class="bg-slate-800 p-2 rounded w-full mr-4"
+              class="  p-2 rounded w-full mr-4"
```
Comment (Israel-Laguan) [inline ui/src/components/MessageContainer.svelte:67]: NIT:
Extra space
```suggestion
              class="p-2 rounded w-full mr-4"
```

File: ui/src/components/MessageInput.svelte [hunk 4]
```diff
@@ -64,24 +65,23 @@
...
     id="send-message-btn"
-    class={`px-4 py-2 rounded w-full mt-2 ${isAgentActive ? "bg-slate-800" : "bg-indigo-700"}`}
+    class={` px-4 py-2 rounded w-20  ${isAgentActive ? "bg-slate-800" : "bg-indigo-700 hover:bg-indigo-600 active:bg-indigo-800"}`}
     on:click={handleSendMessage}
     disabled={isAgentActive}
   >
-    {@html isAgentActive ? "<i>Agent is busy...</i>" : "Send"}
-  </button>
+    <i class="fas fa-paper-plane"></i>
+    <!-- {@html isAgentActive ? "<i>Agent is busy...</i>" : "Send"} -->
```
Comment (Israel-Laguan) [inline ui/src/components/MessageInput.svelte:76]: Why removing this?

File: ui/src/components/MessageInput.svelte [hunk 5]
```diff
@@ -64,24 +65,23 @@
...
     }}
   ></textarea>
-  <div class="token-count absolute right-2 bottom-2 text-gray-400 text-xs">
-    0 tokens
-  </div>
+  
   <button
     id="send-message-btn"
-    class={`px-4 py-2 rounded w-full mt-2 ${isAgentActive ? "bg-slate-800" : "bg-indigo-700"}`}
+    class={` px-4 py-2 rounded w-20  ${isAgentActive ? "bg-slate-800" : "bg-indigo-700 hover:bg-indigo-600 active:bg-indigo-800"}`}
```
Comment (Israel-Laguan) [inline ui/src/components/MessageInput.svelte:71]: NIT:
extra spaces
```suggestion
    class={`px-4 py-2 rounded w-20 ${isAgentActive ? "bg-slate-800" : "bg-indigo-700 hover:bg-indigo-600 active:bg-indigo-800"}`}
```

File: ui/src/components/Sidebar.svelte [hunk 6]
```diff
@@ -1,46 +1,26 @@
 <div
-  class="flex flex-col w-20 bg-slate-950 p-4 space-y-4 items-center shadow-2xl shadow-indigo-700"
+  class="flex flex-col gap-3 border-r border-gray-800 w-fit p-2 space-y-4 items-center "
```
Comment (Israel-Laguan) [inline ui/src/components/Sidebar.svelte:2]: NIT:
Extra space
```suggestion
  class="flex flex-col gap-3 border-r border-gray-800 w-fit p-2 space-y-4 items-center"
```

**Comment by mufeedvh** [comment]
Thank you @Vishnuxx for the PR, this will be merged after some breaking changes have been merged to the UI. Adding the label "implement later" for now. ❤️🙌

**Pull request closed without merging**